app/src/main/java/gov/hhs/cdc/trustedintermediary/wrappers/AuthEngine.java

```diff
@@ -0,0 +1,14 @@
+package gov.hhs.cdc.trustedintermediary.wrappers;
+
+import org.jetbrains.annotations.NotNull;
+
+public interface AuthEngine {
```

Comment:
Can you add a class level JavaDoc describing the purpose?

Reply:
Good call!